### PR TITLE
Adding all the network automation changes and features to the Fluorine release notes

### PR DESCRIPTION
**Pull request by mirceaulinic**
### What does this PR do?

This also adds the docs for two PRs not merged yet: https://github.com/saltstack/salt/pull/48916 and https://github.com/saltstack/salt/pull/48945.

### CODE REVIEW

File: doc/topics/releases/fluorine.rst [hunk 1]
```diff
@@ -80,6 +80,387 @@ orchestration state runner.
 The playbooks modules also includes the ability to specify a git repo to clone
 and use, or a specific directory can to used when running the playbook.
 
+Network Automation
+==================
+
+Beginning with this release, Salt provides much broader support for a variety 
+of network operating systems, and features for configuration manipulation or
+operational commands execution.
```
Comment (rallytime) [inline doc/topics/releases/fluorine.rst]: `commands` --> `command`

File: doc/topics/releases/fluorine.rst [hunk 2]
```diff
@@ -80,6 +80,387 @@ orchestration state runner.
...
+
+NetBox
+------
+
+Added in the previous release, Oxygen, the capabilities of the
```
Comment (rallytime) [inline doc/topics/releases/fluorine.rst]: `Oxygen` --> `2018.3.0`

File: doc/topics/releases/fluorine.rst [hunk 3]
```diff
@@ -80,6 +80,387 @@ orchestration state runner.
...
+:mod:`net.confirm_commit <salt.modules.napalm_network.confirm_commit>`, or
+:mod:`net.cancel_commit <salt.modules.napalm_network.cancel_commit>`, as well 
+as the State functions
+:mod:`netconfig.commit_cancelled <salt.states.netconfig.commit_cancelled>`, or
+:mod:`netconfig.commit_confirmed <salt.states.netconfig.commit_confirmed>` can 
+be used to confirm or cancel a commit.
+
+Please note that the commit confirmed and commit cancelled functionalities are
+available for any platform whether the network devices supports the features 
+natively or not. However, be cautious and make sure you read an understood the
```
Comment (rallytime) [inline doc/topics/releases/fluorine.rst]: `read an understood` --> `read and understand`

File: doc/topics/releases/fluorine.rst [hunk 4]
```diff
@@ -80,6 +80,387 @@ orchestration state runner.
...
+
+Beginning with this release, any NAPALM command executed when
+running under a NAPALM Proxy Minion supports the ``force_reconnect``
+magic argument.
+
+Proxy Minions generally establish a connection with the remote network
+device at the time of the Minion startup and that connection is going to be
+used forever.
+
+If one would need execute a command on the device but connecting using
```
Comment (rallytime) [inline doc/topics/releases/fluorine.rst]: `would need execute` --> `would need to execute`
and
`but connecting` --> `but is connecting`

File: doc/topics/releases/fluorine.rst [hunk 5]
```diff
@@ -80,6 +80,387 @@ orchestration state runner.
...
+
+PeeringDB
+---------
+
+The :mod:`peeringdb <salt.modules.peeringdb>` Execution Module is useful to 
+gather information about other networks you can potentially peer with, and 
+automatically establish BGP sessions, e.g., given just a specific AS number, 
+the rest of the data (i.e., IP addresses, locations where the remote network is 
+available, etc.) is retrieved from PeeringDB, and the session configuration is 
+automated with minimum to no effort (typing the IP addresses manually can be
```
Comment (rallytime) [inline doc/topics/releases/fluorine.rst]: `minimum` --> `minimal`

File: doc/topics/releases/fluorine.rst [hunk 6]
```diff
@@ -80,6 +80,387 @@ orchestration state runner.
...
+PeeringDB
+---------
+
+The :mod:`peeringdb <salt.modules.peeringdb>` Execution Module is useful to 
+gather information about other networks you can potentially peer with, and 
+automatically establish BGP sessions, e.g., given just a specific AS number, 
+the rest of the data (i.e., IP addresses, locations where the remote network is 
+available, etc.) is retrieved from PeeringDB, and the session configuration is 
+automated with minimum to no effort (typing the IP addresses manually can be 
+both tedious and error prone)
```
Comment (rallytime) [inline doc/topics/releases/fluorine.rst]: Need a `.` at the end. :)

**Review by rallytime — COMMENTED** [review]
Thank you so much for adding all of this @mirceaulinic!

I have just some small picky changes. :)

**Comment by mirceaulinic** [comment]
Good finds @rallytime - added https://github.com/saltstack/salt/pull/49007/commits/29b84bb047ca296298178c1f3fee83d2d1d06b03 for fix those.

**Review by rallytime — APPROVED** [review]

**Pull request merged**